src/lib/addKeyDownHandler.ts

```diff
@@ -0,0 +1,44 @@
+import * as _ from 'lodash'
+import keyboardHandlerFilter from './keyboardHandlerFilter'
+import { IAccessibilityDefinition, AccessibilityActions } from 'src/lib/accessibility/interfaces'
+import { IRenderConfigProps } from 'src/lib/renderComponent'
+
+/**
+ * Assigns onKeyDown handler to the Component's part element, based on Component's actions
+ * and keys mappings defined in Accessibility behavior
+ * @param {AccessibilityActions} actions The input element which is to loose focus.
+ * @param {IAccessibilityDefinition} accessibility The input element which is to loose focus.
+ * @param {IRenderConfigProps} props The props which are used to invoke onKeyDown handler passed from top.
+ */
+const addKeyDownHandler = (
+  actions: AccessibilityActions,
+  accessibility: IAccessibilityDefinition,
+  props: IRenderConfigProps,
+) => {
+  accessibility.handlers = {}
```

Comment:
this might lead to a state when global accessibility object is modified. instead of modification, create a copy of the object, for example using
```typescript
const result = {
  {...accessibility},
  hanldlers: {}
}
```

Reply:
Agree, will change